README.md

```diff
@@ -117,3 +71,26 @@ Run!!
 
 Also please check [Mewz](https://github.com/Mewz-project/Mewz.git), a unikernel OS which has WASI interface. 
 ELF file generated by Wasker can be executed on Mewz without any modification.
+
+
+# Development
+
+## Option1 : Use Devcontainer
+You can try Wasker via Devcontainer.
+
+## Option2 : Build from source
+```
+git clone git@github.com:mewz-project/wasker.git
+cd Wasker
+export LLVM_SYS_150_PREFIX=~/.wasker/clang+llvm-15.0.0-x86_64-linux-gnu-rhel-8.4/
+cargo install wasker
+```
+By default, Wasker targets x86_64. If you want to target AArch64, please rewrite [build.rs](./build.rs) and override `LLVM_SYS_150_PREFIX` environment variable as follows.
+
+```rust
+# build.rs
+5: let target = format!("clang+llvm-{}-aarch64-linux-gnu-rhel-8.4", llvm_version);
+
+# shell
+export LLVM_SYS_150_PREFIX=~/.wasker/clang+llvm-15.0.0-aarch64-linux-gnu/
```

Comment:
Is it possible to automatically set the environment variable? By something like `build.rs` or `cargo.toml`?

https://stackoverflow.com/questions/57017066/how-do-i-set-environment-variables-with-cargo
https://doc.rust-lang.org/cargo/reference/build-scripts.html#rustc-env

Reply:
And it would be helpful if `build.rs` renames the LLVM directory to `clang+llvm` regardless of the platforms.